superduperdb/backends/mongodb/query.py

```diff
@@ -604,7 +604,10 @@ def find(this, *args, **kwargs):
                     if args:
                         second_part.append({"$match": args[0] if args else {}})
                     if args[1:]:
-                        second_part.append({'$project': args[1]})
```

Comment:
It looks like this will only work if there's already a projection.

Reply:
I tested this, by adding an additional clause, with an edit, and the results look like this:

```
Document({'_id': ObjectId('653b7b2c3e12acb9a077640a')})
Document({'_id': ObjectId('653b7b2c3e12acb9a0776419')})
Document({'_id': ObjectId('653b7b73123cc8c240193d09')})
Document({'_id': ObjectId('653b7b73123cc8c240193d22')})
Document({'_id': ObjectId('653b7b2c3e12acb9a0776408')})
```